compiler/src/iree/compiler/Dialect/Flow/Transforms/FusionUtils.cpp

```diff
@@ -24,6 +26,11 @@ static llvm::cl::opt<bool> clEnsureInplaceableConsumer(
     llvm::cl::desc("Ensure the consumer is inplaceable for fusion."),
     llvm::cl::init(true));
 
+static llvm::cl::opt<bool> clFuseReductionWithElementwise(
```

Comment:
I wonder if we need a more specific name, fusing reduction with elementwise is already on, the difference here is fusing with a broadcast + elementwise